content/TernaryOperator/TernaryOperator.q.md

```diff
@@ -0,0 +1,903 @@
+# Ternary Operator
+
++++
+
+### Part 1: Sample Code Analysis
+
+:::
+
+/// type=REPL, readonly=true
+
+```php
+<?php
+$studentGrade = 80;
+$remarks = ($studentGrade > 75) ? "Very Good" : "Needs Improvement";
+echo $remarks;
+?>
+```
+/// type=SS, answer=[3]
+
+Execute the program. What is its output?
+
+ - It prints `80`.
+
+ - It prints `$remarks`.
+
+ - It prints `Very Good`.
+
+ - It prints `"Very Good"`.
+
+ - It prints `Needs Improvement`.
+
+
+/// type=MS, answer=[2,4]
+
+Which of the following are variables?
+
+ - `80`
+
+ - `$remarks`
+
+ - `Very Good`
+
+ - `$studentGrade`
+
+ - `$studentGrade > 75`
+
+
+/// type=MS, answer=[3,5]
+
+Which of the following are strings?
+
+ - `80`
+
+ - `$remarks`
+
+ - `Very Good`
+
+ - `$studentGrade`
+
+ - `Needs Improvement`
+
+
+/// type=SS, answer=[3]
+
+In the expression `($studentGrade > 75) ? "Very Good" : "Needs Improvement"` on line 3, what is `?:`?
+
+ - It is a separator.
+
+ - It is a question mark.
+
+ - It is a ternary operator.
+
+ - It is a comparison operator.
+
+ - It is an assignment operator.
+
+
+/// type=SS, answer=[3]
+
+In the expression `($studentGrade > 75) ? "Very Good" : "Needs Improvement"` on line 3, what is `($studentGrade > 75)`?
+
+ - It is an operator.
+
+ - It is a ternary operator.
+
+ - It is a condition operand.
+
+ - It is a value for `true` operand.
+
+ - It is a value for `false` operand.
+
+
+/// type=SS, answer=[4]
+
+In the expression `($studentGrade > 75) ? "Very Good" : "Needs Improvement"` on line 3, what is `"Very Good"`?
+
+ - It is an operator.
+
+ - It is a ternary operator.
+
+ - It is a condition operand.
+
+ - It is a value for `true` operand.
```

Comment:
Maybe we could add `the` between `for` and `true`. Please check, thank you.

> - It is a value for the `true` operand.

![image](https://user-images.githubusercontent.com/57437877/73057578-d372fb80-3ecc-11ea-8e4f-3714365b84cb.png)

Reply:
I don't think we have to follow what is suggested by grammarly since if you try to evaluate the statement an article the is not required since `value for true` is the one that is referred to as the operand and the `a` article is already there. This is one of the exception that grammarly cannot detect which I think we should be careful with. Please check. Thank you.

Reply:
Noted. Thanks.